test/evergreen.yml

```diff
@@ -244,6 +245,25 @@ functions:
             # Save generated documentation
             mv docs docs-$branch
           done
+  "dump stacktraces": &dump_stacktraces
+    command: shell.exec
+    params:
+      working_dir: "wiredtiger/cmake_build"
+      script: |
+        set -o errexit
+        set -o verbose
+        ${python_binary|python3} ../test/evergreen/print_stack_trace.py
+  "upload stacktraces": &upload_stacktraces
+    command: s3.put
+    params:
+      aws_secret: ${aws_secret}
+      aws_key: ${aws_key}
+      local_files_include_filter:
+        - wiredtiger/cmake_build/*stacktrace.txt
+      bucket: build_external
+      permissions: public-read
+      content_type: text/plain
+      remote_file: wiredtiger/${build_variant}/${revision}/artifacts/
```

Comment:
Though it's likely rare, there's a chance two stack trace files generated from the same build variant share the same filename, causing overwrite in the same remote directory. Should task_name and build_id be added to the directory structure or as a filename prefix?

Reply:
It might be a fix that we should do on the `develop` branch first, what do you think?

Reply:
I agree we should fix it on develop branch first.

Reply:
I have created #8879.

Reply:
Aligned with `develop` in c50ed1d46ed41effeb8751eff61a25569e739d3f.